docs/2-Interviews/preparing_for_interviews.md

```diff
@@ -12,22 +13,57 @@ Be prepared to:
 - talk about and work towards writing an algorithm to solve a problem, during the interview
 - you can also expect a “tell me about yourself” question (so be prepared!) and some time where they tell you about their company
 
+### What material could show up in an interview?
+
+This is a difficult question to answer. The more things you know, the less likely it is that an interview question could trip you up. That being said, you have limited time to learn things. There a few concepts that are very likely to show up. There's also a long list of algorithms that _could maybe_ show up once in a while, depending on the company, the interviewer or whether venus and mercury are in the same quadrant of the solar system this week. You could be spending an arbitrarily long time preparing for all possible eventualities.
+
+You can do that if you enjoy it, but it's healthy to set a timebox. That is, allocate yourself a certain amount of time in advance to prepare for interviews. Learn as much as you can during that time, but no more. And in that time, you probably want the biggest bang for the buck.
+
+Very likely to show up in interviews:
+- Understanding O() analysis
+- Arrays vs linked-list
+- Stacks and queues
+- Dictionaries (one of the most common data structures in interviews)
+- Being able to traverse a binary search tree recursively. e.g. search for an element in a BST
+
+The above provide a solid foundation to start with. Some intro-level internships will not ask technical questions at all, or teach you concepts during the interview. For places that do ask technical questions, most will assume that you have a good understanding of these basics.
+
+Likely to show up in interviews at more competitive/intermediate-level internships:
+- Be able to give a 5 minute answer the question "how does the internet work?" or "what happens when I type google.com in the URL bar and press enter?" (see resources)
```

Comment:
I have never been asked this question haha - but have been asked the rest I think - can you move this to the bottom of this list? I think it's also the most scary question (but maybe I'm biased :3)